t/test/mojo.t

```diff
@@ -11,6 +11,8 @@ my $t = Test::Mojo->new;
 subtest 'Basics' => sub {
   isa_ok $t->app, 'Mojolicious', 'right class';
   $t->get_ok('/')->status_is(200)->content_is('Hello Test!');
+  $t->status_like(qr/^2/);
+  $t->status_unlike(qr/^4/);
```

Comment:
Not actual tests.

Reply:
How so? They pass or fail depending on what data is in `$t` and the regex you pass in.

Reply:
All new `Test::Mojo` methods require their own `subtest` section below. With tests for all argument variations.

Reply:
OK